src/alire/alire-utils-switches.ads

```diff
@@ -52,6 +53,11 @@ is
                              Ada83, Ada95, Ada05, Ada12, Ada2022,
                              GNAT_Extensions);
 
+   type Unicode_Support_Kind is
+     (No_Support,  --  Default behavior of GNAT, no switches set (Latin1)
```

Comment:
I don't think those names are meaningful enough.

Reply:
What about `Source_Encoding`/`ASCII`/`Latin_1`/`UTF_8`? (ASCII/Latin_1 would be equivalent -- no `-gnatW8`)